team.py

```diff
@@ -0,0 +1,17 @@
+import user
+
+class Team(user.User):
+    def __init__(self,n):
+        self.name = n
+        self.username = "team"
```

Comment:
Do we want these generic values at all or should they just not be anything to start with? Or possibly we could have the init method just take in a name, username, and password...?